packages/sources/ncfx/src/config/includes.json

```diff
@@ -0,0 +1,398 @@
+[
+  {
+    "from": "AED",
+    "to": "USD",
+    "includes": [
+      {
+        "from": "USD",
+        "to": "AED",
+        "inverse": true
+      }
+    ]
+  },
+  {
+    "from": "BRL",
+    "to": "USD",
+    "includes": [
+      {
+        "from": "USD",
+        "to": "BRL",
+        "inverse": true
+      }
+    ]
+  },
```

Comment:
How did you come up with this list?

Reply:
The NCFX Forex API offers a set list of assets which it sends in every update. I took every instance where the feed was  USD/SYMBOL and gave it an inverse to make it SYMBOL/USD

Reply:
Nice, that works for me. May be nice to add that info in the description somewhere